### PR TITLE
README: Formatting fix, adding jmespath to requirements, adding notes on installing dependencies

### PR DESCRIPTION
**Pull request by mikerenfro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ Requirements
 
 RHEL 7 or CentOS 7 - Other versions are not supported.
 
-`passlib` >= 1.5 on the control node
+`passlib` >= 1.5 on the control node (1.6.2 is in EPEL as `python-passlib`)
+
+`jmespath` on the control node (in EPEL as `python2-jmespath`)
```
Comment (jamescassell) [inline README.md]: ships with RHEL and CentOS

File: README.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ Requirements
 
 RHEL 7 or CentOS 7 - Other versions are not supported.
 
-`passlib` >= 1.5 on the control node
+`passlib` >= 1.5 on the control node (1.6.2 is in EPEL as `python-passlib`)
```
Comment (jamescassell) [inline README.md]: Available in RHEL and CentOS as version 1.6.5-2.el7

**Comment by jamescassell** [comment]
AFAIK, we don't require EPEL for this role.

**Comment by mikerenfro** [comment]
Agreed. Updating now.

**Pull request merged**